radixdlt/src/main/java/com/radixdlt/consensus/EventCoordinator.java

```diff
@@ -58,7 +66,9 @@ public EventCoordinator(
 		Pacemaker pacemaker,
 		VertexStore vertexStore,
 		ProposerElection proposerElection,
-		@Named("self") EUID self
+		QuorumRequirements quorumRequirements,
+		@Named("self") RadixAddress selfAddress,
+		@Named("self") ECKeyPair selfKey
```

Comment:
Shouldn't this be in SafetyRules?

Reply:
This is for signing new-views, so no. But I agree that it should be in a different place, this is what my comments about "unified identity management" were referring to - we're currently implementing the same hash/sign flow in both safety rules and here in EC.